### ISSUE TITLE
CVE-2024-49203 was reported in querydsl-jpa : jar : jakarta in latest version 5.1.0 

### ISSUE THREAD

**Issue by SwapnaAnchuri** [querydsl/querydsl#3759]
CVE-2024-49203 was reported in querydsl-jpa : jar : jakarta in latest version 5.1.0

**Comment by CSIRTTrizna** [comment]
Yes, this vulnerability as well as possible solutions for it is discussed in issue #3757.

**Comment by velo** [comment]
Hi @SwapnaAnchuri ,

First of all, you won't find any help here, project is defunct.

I have patched the issue in both the 6.x and 5.x releases of my fork.

For anyone concerned about this security issue, the fix is available in my fork. To migrate, simply change the group ID from com.querydsl to io.github.openfeign.querydsl.

The fork is fully backward-compatible, so the migration should be seamless.

https://github.com/OpenFeign/querydsl/releases/tag/6.10.1
https://github.com/OpenFeign/querydsl/releases/tag/5.6.1

**Comment by jwgmeligmeyling** [comment]
See https://github.com/querydsl/querydsl/issues/3757
